src/ds/ds6/hw_monitor_extended_buffers.h

```diff
@@ -7,9 +7,17 @@
 
 namespace librealsense
 {
+    class hwm_buffer_type;
     class hw_monitor_extended_buffers : public hw_monitor
```

Comment:
Please add a description for this class.
Why do we need it, what are the limitations of a HMC using XU

Reply:
done